src/InContextSignup.js

```diff
@@ -22,6 +43,26 @@ export class InContextSignup {
         this.isInstalledRecently = incontextSignupDismissedAt.isInstalledRecently
     }
 
+    async openAutofillTooltip () {
+        // Make sure we're working with the latest data
+        await this.device.refreshData()
+
+        // Make sure the tooltip is closed before we try to open it
+        await this.device.uiController?.removeTooltip('stateChange')
+
+        // Make sure the input doesn't have focus so we can focus on it again
+        const activeInput = this.device.activeForm?.activeInput
+        activeInput?.blur()
```

Comment:
🪄 This took me longer than I'd like to admit to figure out -- changing tab doesn't cause the input to blur, which means we can't trigger a focus when we come back after signing-up... unless we first make sure to blur it 😅